app/src/main/java/ru/yandex/practicum/contacts/presentation/filter/model/FilterContactTypeUi.java

```diff
@@ -2,7 +2,10 @@
 
 import androidx.annotation.NonNull;
 
-public class FilterContactTypeUi {
+import ru.yandex.practicum.contacts.presentation.base.ListDiffInterface;
+
+/** @noinspection rawtypes*/
+public class FilterContactTypeUi implements ListDiffInterface {
```

Comment:
Здесь ворнинг можно  пофиксить след образом:
```suggestion
public class FilterContactTypeUi implements ListDiffInterface<FilterContactTypeUi> {
```
Далее см комментарий ниже